README.md

```diff
@@ -127,41 +131,74 @@ To use it:
    * find or create the token-signing key
    * start the web server
 
-   If you are going to be debugging Conjur using `pry.byebug`, you may choose to
-   start the web server by calling `rails server -b 0.0.0.0 webrick` instead of
-   `conjurctl server`. This will allow you to work in the debugger without the
-   server timing out.
-
-4. Cleanup
+   You may choose to debug Conjur using `pry.byebug` or RubyMine IDE. This will 
+   allow you to work in the debugger without the server timing out. To do so, 
+   run the following command instead of `conjurctl server`:
+   - `pry.byebug`: `rails server -b 0.0.0.0 webrick`
+   - RubyMine IDE: `rdebug-ide --port 1234 --dispatcher-port 26162 --host 0.0.0.0 -- bin/rails s -b 0.0.0.0 webrick`
+      - Now that the server is listening, debug the code via RubyMine's debugger.
+   
+1. Cleanup
 
     ```sh-session
     $ ./stop
     ```
     Running `stop` removes the running Docker Compose containers and the data key.
 
-  #### LDAP Authentication
-
-   To enable a user to log into Conjur using LDAP credentials, run `start` with the `--authn-ldap` flag:
+#### LDAP Authentication
 
-   ```sh-session
-   $ cd dev
-   $ ./start --authn-ldap
-   ...
-   root@f39015718062:/src/conjur-server#
-   ```
+To enable a user to log into Conjur using LDAP credentials, run `start` with the `--authn-ldap` flag:
 
-   The `--authn-ldap` flag will:
-  * Start an OpenLDAP container.
-  * Load a user `alice` with the password `alice` into the LDAP server.
-  * Load a policy `authn-ldap/test`, that grants `alice` the ability to authenticate via `http://localhost:3000/authn-ldap/test/cucumber/alice/authenticate` with the password `alice`.
+```sh-session
+$ cd dev
+$ ./start --authn-ldap
+...
+root@f39015718062:/src/conjur-server#
+```
 
+The `--authn-ldap` flag will:
+* Start an OpenLDAP container.
+* Load a user `alice` with the password `alice` into the LDAP server.
+* Load a policy `authn-ldap/test`, that grants `alice` the ability to authenticate via `http://localhost:3000/authn-ldap/test/cucumber/alice/authenticate` with the password `alice`.
 
 Validate authentication using the username `alice` with the password `alice`:
 
 ```sh-session
 $ curl -v -k -X POST -d "alice" http://localhost:3000/authn-ldap/test/cucumber/alice/authenticate
 ```
 
+#### RubyMine IDE Debugging
+
+If you are going to be debugging Conjur using [RubyMine IDE](https://www.jetbrains.com/ruby/), follow
+these steps:
+
+   1. Add a debug configuration
+      1. Go to: Run -> Edit Configurations
+      1. In the Run/Debug Configuration dialog, click + on the toolbar and 
+      choose “Ruby remote debug”
+      1. Specify a name for this configuration (i.e “debug Conjur server”)
+      1. Specify these parameters:
+         - Remote host: localhost
```

Comment:
I would say something like 
"Remote host - the address of Conjur. if it's a local docker environment the address should be localhost, otherwise enter the address of Conjur"